File: tests/fixtures/julia/adapter_matrix.jl
# @generated
module AdapterMatrix

export compute

@generated

function compute(π::Int, values)
    # line comment
    #= block comment =#
    total = π * 2
    message = "total = $total"

    for value in values
        if value > 0
            total += helper(value)
        end
    end

    quoted = quote
        hidden(total)
    end
    @time helper(total)
    return total
end

end

File: tests/fixtures/clojure/adapter_matrix.clj
;; @generated
(ns adapter-matrix)

^:generated
(defn compute [π values]
  ; line comment
  (let [total (* π 2)]
    (if (> total 10)
      (map inc values)
      (do
        '(if quoted (when hidden :bad) :data)
        #_(when discarded :bad)
        #?(:clj (println total) :cljs (js/console.log total))
        `(when ~π ~@values)
        #=(+ 1 2)
        total))))
